tests/Core/Controller/TwoFactorChallengeControllerTest.php

```diff
@@ -20,7 +20,7 @@
  *
  */
 
-namespace Test\Core\Controller;
+namespace Test\Settings\Core\Controller;
```

Comment:
```suggestion
namespace Tests\Core\Controller;
```

why was `Settings` added here?